internal/clientcache/cmd/daemon/start.go

```diff
@@ -104,14 +104,15 @@ func (c *StartCommand) Flags() *base.FlagSets {
 	f.DurationVar(&base.DurationVar{
 		Name:    "refresh-interval",
 		Target:  &c.flagRefreshInterval,
-		Usage:   `Specifies the interval between refresh token supported cache refreshes. Default: 1 minute`,
+		Usage:   `Specifies the interval between refresh token supported cache refreshes. Default: 5 minutes`,
```

Comment:
```suggestion
		Usage:   `Specifies the interval between refresh token supported cache refreshes.`,
```

Reply:
The CLI library automatically includes the default information so you should never write it manually as it's a good way to get out of sync if the default changes.